src/main/java/org/spongepowered/mod/mixin/entity/MixinEntity.java

```diff
@@ -90,6 +92,13 @@
     public abstract void mountEntity(net.minecraft.entity.Entity entityIn);
 
     @Override
+    public UUID getUniqueId() {
```

Comment:
You cannot ```@Override``` this method to implement the method from the interface because after obfuscation it will get the MCP name. Please consult [the mixin examples](https://github.com/SpongePowered/Sponge/blob/master/src/example/java/org/spongepowered/mod/mixin/MixinEntityPlayerExample.java) for how to deal with conflicting methods with the **same** signature/name.